extensions/amp-story/1.0/amp-story.js

```diff
@@ -2322,6 +2320,33 @@ export class AmpStory extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Loads and initializes the amp-story-interactive components if any.
+   * @private
+   */
+  maybeInitializeInteractives_() {
+    const interactives = this.element.querySelectorAll(
+      'amp-story-interactive-quiz, amp-story-interactive-binary-poll, amp-story-interactive-poll'
+    );
+    if (interactives.length > 0) {
+      Services.extensionsFor(this.win)
+        .installExtensionForDoc(
+          this.getAmpDoc(),
+          'amp-story-interactives',
+          '1.0'
+        )
+        .then(() => {
+          toArray(interactives).forEach((element) => {
+            whenUpgradedToCustomElement(element).then((el) =>
+              el.getImpl().then((unusedImpl) => {
+                // TODO(mszylkowski): Initialize impl
+              })
+            );
+          });
+        });
```

Comment:
Two options here:
1. Keep the method call you have in the interactive constructor and remove this block
2. Somehow extract the method from the interactive component and have it as a `util` function that you can call here, without having to get the `impl` of the component. All it uses is the `interactiveID`, which we can generate from a helper.

Both are fine with me, (1) is easier

Reply:
Ok, 1 sounds good in this case

Reply:
Sounds good, let's remove this block then?